Adyen/UI/Form/Items/Error/FormErrorItemView.swift

```diff
@@ -19,7 +19,7 @@ internal final class FormErrorItemView: FormItemView<FormErrorItem> {
         bind(item.$message, to: self, at: \.accessibilityLabel)
         isHidden = item.isHidden.wrappedValue
         addSubview(containerView)
-        containerView.adyen.anchor(inside: layoutMarginsGuide, with: UIEdgeInsets(top: 16, left: 16, bottom: -16, right: -16))
+        containerView.adyen.anchor(inside: layoutMarginsGuide, with: UIEdgeInsets(top: 8, left: 0, bottom: -8, right: 0))
```

Comment:
It would be great to see all UI changes in the code supported with screenshots

Reply:
Will be once CI works again (Need to run snapshot updates)